src/protocol/mod.rs

```diff
@@ -50,6 +50,11 @@ pub struct WebSocketConfig {
     /// be reasonably big for all normal use-cases but small enough to prevent memory eating
     /// by a malicious user.
     pub max_frame_size: Option<usize>,
+    /// If set to true it will allow the websocket server to accept unmasked frames from client.
+    /// Even though this behaviour is not in compliance with RFC 6455 (which requires the server
+    /// to close the connection when unmasked frame from client is received) it might be handy in some cases
+    /// as there are existing applications sending unmasked client frames.
```

Comment:
Nit: we should probably mention that default is false. (it's kinda obvious, but documentation should probably state even the obvious things)

Reply:
documentation improved.